native/cocos/network/Downloader-curl.cpp

```diff
@@ -234,6 +234,15 @@ class DownloaderCURL::Impl : public enable_shared_from_this<DownloaderCURL::Impl
     //        : _thread(nullptr)
     {
         DLLOG("Construct DownloaderCURL::Impl %p", this);
+        #if (CC_PLATFORM == CC_PLATFORM_NX)
```

Comment:
Please add comments about why need to do special operation on NX.

Reply:
In addition to the NX platform, there will be no crashes on other platforms. Not sure if it's a version issue or something else